packages/electron-filestore/lib/minidump-io.js

```diff
@@ -27,7 +27,12 @@ const getIdentifier = (filepath) => {
     let keyIndex = -1
 
     const extractValue = () => {
-      const start = keyIndex + byteOffset
+      // in this case we have a key embedded in an multipart form field (0x22 === '\"')
+      const formEncoded = data[keyIndex - 1] === 0x22 || data[keyIndex + lookupKeyLength] === 0x22
```

Comment:
Do we need both of these checks? The tests pass if either one is removed

Reply:
I would prefer to keep these as their codes directly to avoid any possible confusion about encodings, but naming the values definitely adds value 👍  How about:
```
// ASCII / UTF-8 byte value for a double-quote '"' character
const DOUBLE_QUOTE = 0x22
```

As for having both checks, the idea was to be forgiving rather than strict on this check. Any malformed data will be picked up further down by
```validate(identifier)```

Reply:
`const DOUBLE_QUOTE = 0x22` LGTM :+1: